docs/resource/host.md

```diff
@@ -12,45 +12,31 @@ To report host metrics, the `system.*` namespace SHOULD be used.
+| [`host.image.name`](../attributes-registry/host.md) | string | Name of the VM image or OS install the host was instantiated from. | `infra-ami-eks-worker-node-7d4ec78312`; `CentOS-8-x86_64-1905` | Recommended |
+| [`host.image.version`](../attributes-registry/host.md) | string | The version string of the VM image or host OS as defined in [Version Attributes](../resource/README.md#version-attributes). | `0.1` | Recommended |
+| [`host.ip`](../attributes-registry/host.md) | string[] | Available IP addresses of the host, excluding loopback interfaces. [1] | `[192.168.1.140, fe80::abc2:4a28:737a:609e]` | Recommended |
+| [`host.mac`](../attributes-registry/host.md) | string[] | Available MAC addresses of the host, excluding loopback interfaces. [2] | `[AC-DE-48-23-45-67, AC-DE-48-23-45-67-01-9F]` | Recommended |
+| [`host.type`](../attributes-registry/host.md) | string | Type of host. For Cloud, this must be the machine type. | `n1-standard-1` | Recommended |
 
 **[1]:** IPv4 Addresses MUST be specified in dotted-quad notation. IPv6 addresses MUST be specified in the [RFC 5952](https://www.rfc-editor.org/rfc/rfc5952.html) format.
 
 **[2]:** MAC Addresses MUST be represented in [IEEE RA hexadecimal form](https://standards.ieee.org/wp-content/uploads/import/documents/tutorials/eui.pdf): as hyphen-separated octets in uppercase hexadecimal form from most to least significant.
-
-`host.arch` has the following list of well-known values. If one of them applies, then the respective value MUST be used, otherwise a custom value MAY be used.
-
-| Value  | Description |
-|---|---|
-| `amd64` | AMD64 |
-| `arm32` | ARM32 |
-| `arm64` | ARM64 |
-| `ia64` | Itanium |
-| `ppc32` | 32-bit PowerPC |
-| `ppc64` | 64-bit PowerPC |
-| `s390x` | IBM z/Architecture |
-| `x86` | 32-bit x86 |
 <!-- endsemconv -->
 
 **type:** `host.cpu`
 
 <!-- semconv host.cpu -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
-| `host.cpu.cache.l2.size` | int | The amount of level 2 memory cache available to the processor (in Bytes). | `12288000` | Opt-In |
-| `host.cpu.family` | int | Numeric value specifying the family or generation of the CPU. | `6` | Opt-In |
-| `host.cpu.model.id` | int | Model identifier. It provides more granular information about the CPU, distinguishing it from other CPUs within the same family. | `6` | Opt-In |
-| `host.cpu.model.name` | string | Model designation of the processor. | `11th Gen Intel(R) Core(TM) i7-1185G7 @ 3.00GHz` | Opt-In |
-| `host.cpu.stepping` | int | Stepping or core revisions. | `1` | Opt-In |
-| `host.cpu.vendor.id` | string | Processor manufacturer identifier. A maximum 12-character string. [1] | `GenuineIntel` | Opt-In |
+| [`host.cache.l2.size`](../attributes-registry/host.md) | int | The amount of level 2 memory cache available to the processor (in Bytes). | `12288000` | Opt-In |
```

Comment:
Are these changes intentional? I think not, right?

Reply:
I have moved host.cpu namespace to the host registry as well. So I think this should be correct.